build-plugin/build.gradle.kts

```diff
@@ -18,6 +18,8 @@ dependencies {
 
     implementation(libs.diff.utils)
     compileOnly(libs.android.tools.common)
+
+    implementation(platform(libs.kotlin.gradle.bom))
```

Comment:
It this really necessary? My local build works fine without this line.

If we don't need it, we could get rid of `kotlinGradleBom` in `libs.versions.toml`. That would be one thing less to manually change when updating the Gradle version.

Reply:
Gradle ships with a dedicated version of Kotlin Script and forcing another Kotlin version could work when there are patch version differences, but will fail when there is a major or minor version difference.

Reply:
We actually don't need that check, I just put it there for you, as you told me that you don't want to have too many versions visible int the external libs view of Android Studio. So see this as a service. It's absolutely not necessary.

Reply:
Nice :+1:

I think it would be good to have this information in a comment. Without this information, I think it's hard to deduce why we're doing this.